ui/desktop/app/routes/scopes.js

```diff
@@ -6,14 +6,35 @@
 import Route from '@ember/routing/route';
 import { A } from '@ember/array';
 import { inject as service } from '@ember/service';
+import { set, get } from '@ember/object';
 
 export default class ScopesRoute extends Route {
   // =services
 
   @service store;
+  @service ipc;
 
   // =methods
 
+  async beforeModel() {
+    let isClientDaemonSupported = true;
```

Comment:
This variable isn't quite accurate. We're really checking to see if the controller supports pagination and we're inferring if client daemon is supported by that. In the future, we can handle not having the client daemon up but we will always need pagination.

Reply:
Updated to `isPaginationSupported`. Thanks for the feedback.